individual/gql_mutations.py

```diff
@@ -323,4 +332,39 @@ def _mutate(cls, user, **data):
         return result if not result['success'] else None
 
     class Input(CreateGroupInputType):
-        individual_ids = graphene.List(graphene.UUID)
+        individual_ids = graphene.List(graphene.UUID, required=True)
+
+
+class CreateGroupAndMoveIndividualMutation(BaseHistoryModelCreateMutationMixin, BaseMutation):
+    _mutation_class = "CreateGroupAndMoveIndividualMutation"
+    _mutation_module = "individual"
+    _model = Group
+
+    @classmethod
+    def _validate_mutation(cls, user, **data):
+        super()._validate_mutation(user, **data)
+
+        required_perms = [
+            IndividualConfig.gql_group_create_perms,
+            IndividualConfig.gql_group_update_perms
+        ]
+
+        if not any(user.has_perms(perm) for perm in required_perms):
+            raise ValidationError("mutation.authentication_required")
+
+    @classmethod
+    def _mutate(cls, user, **data):
+        if "client_mutation_id" in data:
+            data.pop('client_mutation_id')
+        if "client_mutation_label" in data:
+            data.pop('client_mutation_label')
+
+        service = CreateGroupAndMoveIndividualService(user)
+        if IndividualConfig.gql_check_group_individual_update:
```

Comment:
what is a difference between update and create? I see you create task in update mode only? And what about 'create' mode (new group)?

Reply:
Usually I see here that task is performed when 'config' for task is turn on. (marked as true).

Reply:
I changed that to if update group individual tasks or group create tasks turned on